runtime/vara/src/migrations.rs

```diff
@@ -14,4 +15,4 @@ impl OnRuntimeUpgrade for SessionValidatorSetMigration {
     }
 }
 
-pub type Migrations = (SessionValidatorSetMigration,);
+pub type Migrations = (SessionValidatorSetMigration, GasTreeMigration<Runtime>);
```

Comment:
Why should we specify all migrations in runtime directly?

Reply:
The block executor needs an implementation of the `OnRuntimeUpgrade` trait to apply migrations